image.css

```diff
@@ -0,0 +1,29 @@
+.image_group {
+    background-color: #F5F7F9;
+    min-height: 100px;
```

Comment:
You can use rem instead of pixels over here.

Reply:
Done